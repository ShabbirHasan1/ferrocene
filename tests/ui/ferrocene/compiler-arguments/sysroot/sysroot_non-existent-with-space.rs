// Test when --sysroot set the path to a non-existent sysroot
//
// check-fail
// ignore-cross-compile The error message is different when cross-compiling
// compile-flags: --sysroot /non/existent/sysroot
//~^^^^^ ERROR can't find crate for `std`

fn main() {}

// ferrocene-annotations: um_rustc_sysroot
