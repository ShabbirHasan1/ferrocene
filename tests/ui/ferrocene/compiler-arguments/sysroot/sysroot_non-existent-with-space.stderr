error[E0463]: can't find crate for `std`

error: requires `sized` lang_item

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0463`.
